.github/workflows/clarity-js-sdk-pr.yml

```diff
@@ -11,12 +11,13 @@ env:
 on:
   release:
     types:
-      - published
+      - released
```

Comment:
What exactly is the difference between the `released` event and the `published` event in GitHub actions?

Reply:
The `released` type will trigger when a new release is created, the `published` type will trigger when a new release **or pre-release** is published. The difference between created vs published isn't so important here, but I don't want this job running when new pre-releases are created. I've been using `released` in `clarinet` and it seems to work much better!